doc/neogit.txt

```diff
@@ -100,8 +100,18 @@ p                       Open pull popup
 
                                                 *neogit_P*
 P                       Open push popup
+
                                                 *neogit_f*
 f                       Open fetch popup
+
```

Comment:
Fantastic job on keeping the documentation up to date :+1: 

It is usually the part most forgotten about